CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.3.0...HEAD)
+## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.3.1...HEAD)
+
+## [0.3.1](https://github.com/fivetran/go-fivetran/compare/v0.3.0...v0.3.1) - 2021-12-08
```

Comment:
It looks like this should be `0.4.0` and not `0.3.1` - https://semver.org/spec/v2.0.0.html

Reply:
Fixed